Automattic/wc-calypso-bridge
# Add bridge notes and cron events

Fixes https://github.com/Automattic/wc-calypso-bridge/issues/626

* Adds cron events under the action `wc_calypso_bridge_daily` event name.
* Adds the navigation learn more note.

Up for discussion on whether this belongs here or not.  The events and notes give us a finer level of control inside the bridge, but obviously come with some level of overhead.

TODO: The CTA link still needs to be replaced.

### Testing

1. Make sure you're under dev mode for the ecomm plan.
2. Toggle the navigation on/off and make sure the note is added when it's turned on.
3. Delete the note while the nav is already turned on.
4. Run the `wc_calypso_bridge_daily` event.
5. Make sure the note is added.
